src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java

```diff
@@ -75,6 +76,14 @@
   @Macro
   private String offset;
 
+  @Name(SalesforceSourceConstants.PROPERTY_OPERATION)
+  @Description("If set to query, the query result will only return current rows. If set to queryAll, " +
```

Comment:
nit: Is there a way to highlight `query` and `queryAll` to indicate that it is a keyword?